Examples/BookingSystem.AspNetCore/Stores/FacilityStore.cs

```diff
@@ -263,60 +264,59 @@ protected async override Task GetOrderItems(List<OrderItemContext<FacilityOpport
 
             // Response OrderItems must be updated into supplied orderItemContexts (including duplicates for multi-party booking)
 
-            using (var db = FakeBookingSystem.Database.Mem.Database.Open())
+            var query = orderItemContexts.Select((orderItemContext) =>
```

Comment:
This has been refactor to call a function on FakeDatabase instead of manipulating the tables from here